### PR TITLE
I corrected some typos in the documentation :)

### PR DESCRIPTION
**Pull request by tusharhero**
Accept this pull request to correct the errors without grammarly

### CODE REVIEW

File: guides/Installation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 🛠️ Install the app
 > ## NOTE:  
-> You should have [Node](https://nodejs.org) and [npm](https://www.npmjs.com/) installed in your PC in order to install this app and run it.
+> You should have [nodejs](https://nodejs.org) and [npm](https://www.npmjs.com/) installed in your PC in order to install this app and run it.
```
Comment (biologyscience) [inline guides/Installation.md]: let it be **Node.js**

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # OverTone
-A free and open source Music Player for all your local music files !
+A free and open-source Music Player for all your local music files !
```
Comment (biologyscience) [inline README.md]: let this be **open source** itself

**Comment by tusharhero** [comment]
Done!

**Comment by biologyscience** [comment]
Thank you for your contribution 😄

**Pull request merged**